.github/workflows/helm_charts_scalar.yml

```diff
@@ -12,27 +12,27 @@ on:
   workflow_dispatch:
 
 env:
-  HELM_VERSION: v3.5.2
+  HELM_VERSION: v3.11.3
 
 jobs:
   lint-chart:
     runs-on: ubuntu-latest
     env:
-      PYTHON_VERSION: 3.7
+      PYTHON_VERSION: 3.11
```

Comment:
Based on the Python document, I chose python v3.11 which is the latest supported version.
https://devguide.python.org/versions/